charts/ratify/README.md

```diff
@@ -48,8 +48,9 @@ Values marked `# DEPRECATED` in the `values.yaml` as well as **DEPRECATED** in t
 | tolerations                                        | Pod tolerations for the Ratify deployment                                                                                                                                                                                                                                                                                                                              | `[]`                              |
 | notationCerts                                      | An array of public certificate/certificate chain used to create inline certstore used by Notation verifier                                                                                                                                                                                                                                                             | ``                                |
+| cosignKeys                                         | An aray of public keys used to create inline key management providers used by Cosign verifier                                                                                                                                                                                                                                                                          | `[]`                              |
 | cosign.enabled                                     | Enables/disables cosign tag-based signature lookup in ORAS store. MUST be set to true for cosign verification.                                                                                                                                                                                                                                                         | `true`                            |
-| cosign.key                                         | Public certificate used by cosign verifier                                                                                                                                                                                                                                                                                                                             | ``                                |
+| cosign.scopes                                      | An array of scopes relevant to the single trust policy configured in Cosign verifier                                                                                                                                                                                                                                                                                   | `["*"]`                           |
```

Comment:
need to add link to docs to explain supported scopes syntax once docs are published on website